bio/rsem/prepare-reference/meta.yaml

```
@@ -1,12 +1,13 @@
 name: rsem prepare reference
 description: |
         Run rsem-prepare-reference to create index files for downstream analysis with rsem.
+url: https://github.com/deweylab/RSEM
 authors:
   - Brett Copeland
 input:
-  - reference genome
-  - additional optional arguments
+  - ref: reference FASTA with either the entire genome or transcript sequences. 
+      If the entire genome is used, the --gtf option must be specified under params.
 output:
-  - index files for downstream use with rsem
-notes: |
-  * For more information, see https://github.com/deweylab/RSEM.
+  - index files for downstream usage with RSEM
```

The name of the output file has to be `seq`, right?
```suggestion
  - seq: index files for downstream usage with RSEM
```

---

I am not one of the RSEM developers, but in the example code (Snakefile) they had just written, that 
```
# one of the index files created and used by RSEM (required)
seq="index/reference.seq",
```

So, in general all the listed output files are part of the file set that is produced for indexing purpose.